FSQCellManifest/FSQCellManifest.h

```diff
@@ -129,7 +129,7 @@ extern const NSInteger kRowIndexForFooterIndexPaths;
  Setting this property is equivalent to calling `setSectionRecords:selectionStrategy:`
  with a selection strategy of FSQViewReloadCellSelectionStrategyDeselectAll.
  */
-@property (nonatomic) NSArray<FSQSectionRecord *> *sectionRecords;
+@property (nonatomic, null_resettable) NSArray<FSQSectionRecord *> *sectionRecords;
```

Comment:
Does this correctly behave in implementation as `null_resettable`? I.E. will this property be `@[]` after you set it to `nil`?

Reply:
Following the code is a bit ugly here but ultimately setSectionRecords calls: [self setSectionRecords: selectionStrategy]
which calls: [self replaceSectionRecords:sectionRecords selectionStrategy:selectionStrategy managedViewUpdates:nil]
which ends up going to this line: https://github.com/foursquare/FSQCellManifest/blob/master/FSQCellManifest/FSQCellManifest.m#L271